app/templates/analytics.txt

```diff
@@ -1 +1 @@
-GooGhywoiu9839t543j0s7543uw1 - pls add steven.bagley@nhs.net to GA account UA-137452494 with ‘Manage Users and Edit’ permissions - date 8 December 2022.
\ No newline at end of file
+GooGhywoiu9839t543j0s7543uw1.  Please add steven.bagley@nhs.net to GA account 137452494 with “Administrator” permissions - date 16th December 2022.
```

Comment:
I don't know how picky Google is about this format, but there are two spaces after the `.`. If there's a human in the loop with this process then it certainly isn't worth changing.

Reply:
I assume that given the google help for analytics.txt also has two spaces at this point that this should be ok ;)